physicsnemo/models/module.py

```diff
@@ -552,8 +736,8 @@ class attribute. Attempting to override any other argument will raise
 
     @staticmethod
     def from_torch(
-        torch_model_class: torch.nn.Module, meta: ModelMetaData = None
-    ) -> "Module":
+        torch_model_class: type[torch.nn.Module], meta: ModelMetaData | None = None
```

Comment:
Was the lack of `type[...]` the bug you mentioned? Seems like this flat out wouldn't have worked before if that is needed here...